PWGCF/Femto3D/DataModel/singletrackselector.h

```diff
@@ -479,6 +483,10 @@ DECLARE_SOA_TABLE(SingleTrkExtras, "AOD", "SINGLETRKEXTRA",
                   singletrackselector::TPCSignal,
                   singletrackselector::Beta);
 
+DECLARE_SOA_TABLE(SinglePIDEls, "AOD", "SINGLEPIDEL",
+                  singletrackselector::StoredTPCNSigmaEl,
+                  singletrackselector::TPCNSigmaEl<singletrackselector::StoredTPCNSigmaEl>);
+
```

Comment:
Why a separated table?
Derived data will be produced together with the other tables and as such has to be coordinated
Should not be then better to have it as a natural extension of the other table which stores the same information about Pi, Ka, ...?

Reply:
Hi @victor-gonzalez , this is the first step towards the splitting of PID tables for different types of particles in order to use them as joinable. Since we needed to urgently add this new column we decided to move already in that direction.